Production_scripts/B07_Process_District_Nursing_extract.R

```diff
@@ -0,0 +1,196 @@
+#####################################################
+# Districy Nursing Extract
+# Author: Catherine Holland
+# Date: February 2022
+# Written on RStudio Server
+# Version of R - 3.6.1
+# Input -
+# Description - Process District Nursing Extract
+#####################################################
+
+# Load packages
+library(dplyr)
+library(ggplot2)
+library(createslf)
+library(slfhelper)
+library(phsmethods)
+library(lubridate)
+
+
+# Read in data ---------------------------------------
+
+# Specify year
+year <- 1920
```

Comment:
```suggestion
year <- '1920'
```
I think we need to specify year as a character

Reply:
```suggestion
year <- check_year_format("1920")
```